packages/cursorless-vscode/scripts/install-local.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Bundles and installs a local version of Cursorless, uninstalling production
+# Cursorless first and using a special extension id to break update chain
+
+# 1. Uninstall production cursorless
+code --uninstall-extension pokey.cursorless || echo "Cursorless not currently installed"
```

Comment:
`--disable-extension`

Reply:
Turns out this flag doesn't do what we thought it did; it just tries to launch vscode with the extension disabled; it doesn't disable the extension everywhere.  Leaving it as uninstall cc/ @AndreasArvidsson